sw/device/tests/sim_dv/i2c_host_tx_rx_test.c

```diff
@@ -86,6 +86,9 @@ void ottf_external_isr(void) {
       done_irq_seen = true;
       i2c_irq = kDifI2cIrqCmdComplete;
       break;
+    // TODO: remove after #17811 is fixed
+    case kDifI2cIrqSdaInterference:
```

Comment:
FYI: With CDC instrumentation enabled, the SDA interference detection ostensibly gets false triggers. You see, an added cycle to the prim_flop_2sync is applied incorrectly when I2C is driving SDA in host mode: In this case, there is no clock domain crossing, as all flops and signals are timed in the `io_clk_div4` domain.

However, this might not be so true in the real ASIC, where the round-trip delay through pads may be more than one clock cycle! (Open-drain, I2C-oriented pads can be quite a bit slower than straightforward push-pull types.)

Reply:
If this(https://cs.opensource.google/opentitan/opentitan/+/master:hw/ip/i2c/dv/env/i2c_seq_cfg.sv;drc=0a9e884f0f9b4655fefadf823c485c003c014834;l=58) is correct,
I see t_r has some ranges. So maybe we should create an errata after we try on fpga
or GLS?